README.md

```diff
@@ -1 +1,39 @@
-# My-Enumerable
\ No newline at end of file
+# My-Enumerable
+Here in this project I have created a class named `MyList` and a module named my enumerable. The module `MyEnumerable` will implement a subset of the functionality of [Enumerable](https://ruby-doc.org/core-3.0.0/Enumerable.html)
+
+## Shot Demo
+![](/assets/my_enumerable.png)
+
+
+## Project requirements
+Create a class `MyList` that has an instance variable `@list`.
+In `MyList` implement a method `#each` that yields successive members of `@list` and uses the `MyEnumerable` module.
+
+...you can see more of the requirements [here](https://github.com/microverseinc/curriculum-ruby/blob/main/simple-ruby/create_your_own_enumerable.md)
+
+## Built with
+Programming language: `Ruby`
+Frameworks: `NaN`
+Technology: `NaN`
+Testing tool: [Rubocup](https://rubocop.org/)
```

Comment:
- [ ] I would recommend including the steps to clone and run the project locally to make the README professional.

```
git clone https://github.com/oluyaratosin123/My-Enumerable.git
cd My-Enumerable
ruby my_list.rb
```